packages/client/mutations/CreateReflectionMutation.ts

```diff
@@ -58,7 +65,30 @@ const CreateReflectionMutation: StandardMutation<TCreateReflectionMutation> = (
   return commitMutation<TCreateReflectionMutation>(atmosphere, {
     mutation,
     variables,
-    onCompleted,
+    onCompleted: (res, errors) => {
+      const draftKudoses = res.createReflection?.draftKudoses
+      if (draftKudoses && draftKudoses.length) {
+        const preferredNames = draftKudoses
+          .map((kudos) => kudos.receiverUser.preferredName)
+          .join(', ')
+        atmosphere.eventEmitter.emit('addSnackbar', {
+          key: 'youGaveKudos',
```

Comment:
+1 If I give kudos in 2 different reflections within 5s, I will not see the second notification because they share the same key. This is an edge case unlikely to ever occur